var/spack/repos/builtin/packages/nginx/package.py

```diff
@@ -14,13 +14,16 @@ class Nginx(AutotoolsPackage):
     homepage = "https://nginx.org/en/"
     url      = "https://nginx.org/download/nginx-1.12.0.tar.gz"
 
+    version('1.15.6', 'a3d8c67c2035808c7c0d475fffe263db8c353b11521aa7ade468b780ed826cc6')
     version('1.13.8', 'df4be9294365782dc1349ca33ce8c4ac')
     version('1.12.0', '995eb0a140455cf0cfc497e5bd7f94b3')
 
     depends_on('openssl')
     depends_on('pcre')
     depends_on('zlib')
 
+    conflicts('%gcc@8:', when='@:1.14')
```

Comment:
there is a msg attribute, you could use it to document that this is based on "all spack knows" rather than on "that's what they claim", but no strong opinions

Reply:
I just checked out the nginx source again and the fix is not on the 1.14 branch but in 1.15.0, so this conflicts should actually be correct. :-)